File: server/src/main/java/com/krillsson/sysapi/core/domain/metadata/MetaMapper.java
package com.krillsson.sysapi.core.domain.metadata;

import org.mapstruct.Mapper;
import org.mapstruct.ReportingPolicy;
import org.mapstruct.factory.Mappers;

@Mapper(
        unmappedTargetPolicy = ReportingPolicy.ERROR
)
public interface MetaMapper {
    MetaMapper INSTANCE = Mappers.getMapper(MetaMapper.class);

    com.krillsson.sysapi.dto.metadata.Meta map(com.krillsson.sysapi.core.domain.metadata.Meta value);
}

File: server/src/main/java/com/krillsson/sysapi/core/domain/metadata/MetaMapper.kt
package com.krillsson.sysapi.core.domain.metadata

import org.mapstruct.Mapper
import org.mapstruct.ReportingPolicy
import org.mapstruct.factory.Mappers

@Mapper(unmappedTargetPolicy = ReportingPolicy.ERROR)
interface MetaMapper {
    fun map(value: Meta): com.krillsson.sysapi.dto.metadata.Meta

    companion object {
        @kotlin.jvm.JvmField
        val INSTANCE = Mappers.getMapper(MetaMapper::class.java)
    }
}